src/main/java/uk/gov/ons/census/casesvc/utility/FieldworkHelper.java

```diff
@@ -3,17 +3,46 @@
 import uk.gov.ons.census.casesvc.model.entity.Case;
 
 public class FieldworkHelper {
-  public static boolean shouldSendCaseToField(Case caze, String eventChannel) {
-    if (eventChannel.equals("FIELD")) {
+  public static boolean shouldSendUpdatedCaseToField(Case caze, String eventChannel) {
+    if (hasChannelField(eventChannel)) {
       return false;
     }
-    if (caze.getRegion().startsWith("N") && caze.getCaseType().equals("CE")) {
+
+    if (hasRegionNorthernIrelandAndCECaseType(caze)) {
       return false;
     }
+
     if (caze.getEstabType().equals("TRANSIENT PERSONS")
         || caze.getEstabType().equals("MIGRANT WORKERS")) {
       return false;
     }
     return true;
   }
+
+  public static boolean shouldSendUnInvalidatedAddressCaseToField(Case caze, String eventChannel) {
+    if (hasChannelField(eventChannel)) {
+      return false;
+    }
+
+    if (hasRegionNorthernIrelandAndCECaseType(caze)) {
+      return false;
+    }
+
+    if (caze.getRefusalReceived() != null) {
+      return false;
+    }
+
+    if (caze.getEstabType() != null) {
+      return !caze.getEstabType().equals("TRANSIENT PERSONS");
```

Comment:
so we _do_ want to send these to `MIGRANT WORKERS`? That seems oddly specific

Reply:
Yeah, it does seem odd that the acceptance criteria is different here. Will double-check with Dan 👍